Mu2eUtilities/inc/MueXSpectrum.hh

```diff
@@ -0,0 +1,58 @@
+#ifndef Mu2eUtilities_MueXSpectrum_hh
+#define Mu2eUtilities_MueXSpectrum_hh
+
+// CLHEP includes
+#include "CLHEP/Vector/LorentzVector.h"
+
+// C++ includes
+#include <utility>
+
+namespace mu2e {
+
+  double f(double E, void *p);
+
+  class MueXSpectrum {
+
+  public:
+    
+    struct Params_t {
```

Comment:
Not a big fan of a giant struct declared inside the class, especially since most of these numbers are hardcoded later. Do you really need all this? If so, why not make them class member?

Reply:
Hi Bertrand, I followed the same structure as the CE spectrum, this is how they do it in that file.